HTML/Ultimate Cipher condensed (Fang).html

```diff
@@ -0,0 +1,1861 @@
+﻿<!DOCTYPE html>
+<html>
+
+<head>
+	<meta http-equiv="content-type" content="text/html; charset=UTF-8">
+	<meta charset="utf-8">
+	<title>The Ultimate Cipher - Keep Talking and Nobody Explodes Module</title>
+	<meta name="viewport" content="initial-scale=1">
+	<link rel="stylesheet" type="text/css" href="css/normalize.css">
+	<link rel="stylesheet" type="text/css" href="css/main.css">
+	<link rel="stylesheet" type="text/css" href="css/font.css">
+	<script src="js/ktane-utils.js"></script>
+	<style>
+        table {
+            margin: 1em auto;
+        }
+        .strip-change-one td,
+        .strip-change-one th {
+            padding: .3em .5em .2em;
+        }
+        .tablea {
+            font-size: 14px;
+        }
+		.tableb {
+            font-size: 20px;
+			   width: 100%;
+		}
+		.alpha {
+            font-size: 13px;
+        }
+	.rotor {
+            font-size: 13px;
+        }
+	.reflector {
+            font-size: 14px;
+        }
+	.alpha {
+            font-size: 13px;
+        }
+	.beta {
+            font-size: 11px;
+        }
+
+
+    </style>
+	<style>
+        /* Interactive */
+        .borderless {
+            border: none;
+        }
+
+        #interactive {
+            font-size: 12.5px;
+        }
+		#interactive1 {
+            font-size: 10px;
+        }
+        
+        #interactive td.borderless:first-child {
+            padding: .3em;
+        }
+
+		button {
+			font-family: Special Elite;
+			background-color: white;
+			color: black;
+			padding: 1px 3px;
+			border-radius: 5px;
+			border: black 2px solid;
+			transition: 0.3s;
+		}
+
+		button:hover {
+			background-color: black;
+			color: white;
+		}
+    </style>
+    <script src="js/jquery.3.1.1.min.js"></script>

```

Comment:
if this is interactive, please mark this as such

Reply:
he said condensed thou, so i did both interactive and condensed, it is pretty much condensed lol